src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+
+html,
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Not every this tag has this default styles, better to reset styles for particular tags instead
And `box-sizing` rule is not inherited, so there is no sense to set it to html and body tags